packages/fxa-settings/src/lib/cache.ts

```diff
@@ -3,36 +3,33 @@ import Storage from './storage';
 import { Email } from '../models';
 import { searchParam } from '../lib/utilities';
 import config from './config';
+import { StoredAccountData } from './storage-utils';
 
 const storage = Storage.factory('localStorage');
 
-export interface OldSettingsData {
-  uid: hexstring;
-  sessionToken: hexstring;
-  alertText?: string;
-  displayName?: string;
-  metricsEnabled?: boolean;
-  email?: string;
-}
-
-export function getOldSettingsData({
+export function getStoredAccountData({
   uid,
   sessionToken,
   alertText,
   displayName,
   metricsEnabled,
-}: Record<string, any>): OldSettingsData {
+  lastLogin,
+  email,
+  verified,
+}: Record<string, any>): StoredAccountData {
   return {
     uid,
     sessionToken,
     alertText,
     displayName,
     metricsEnabled,
+    lastLogin,
+    email,
+    verified,
   };
 }
 
-type LocalAccount = OldSettingsData | undefined;
-type LocalAccounts = Record<hexstring, LocalAccount> | undefined;
+type LocalAccounts = Record<hexstring, StoredAccountData> | undefined;
```

Comment:
Nit: I feel like having the | undefined here is a bit weird. We can just use the the optional '?' on the function param, which seems to be currently used anyways.